pkg/config/autodiscover/podset_info.go

```diff
@@ -61,27 +57,27 @@ type DeploymentResource struct {
 }
 
 // GetName returns deployment's metadata section's name field.
```

Comment:
"... returns deployment/statefulset's...". Please update all function's comments of this and other files to add the "/statefulset".